style.css

```diff
@@ -1,15 +1,15 @@
 @font-face {
-    font-family: 'Lato';
+    font-family: 'univers';
     /* There are a ton of font options on Google Fonts */
     src: url(http://fonts.googleapis.com/css?family=Lato);
 }
```

Comment:
You also need to update the src url if you want to change the font- this throws an error (as those names have to match) and uses the browser default font